File: admoblibrary/src/main/java/com/rikatech/admoblibrary/callback/BannerAdCallback.java
package com.rikatech.admoblibrary.callback;

import com.google.android.gms.ads.AdSize;
import com.google.android.gms.ads.AdValue;

public interface BannerAdCallback {
    void onBannerAdLoaded(AdSize adSize);
    void onAdFail();
    void onAdPaid(AdValue adValue);
}

File: admoblibrary/src/main/java/com/rikatech/admoblibrary/callback/BannerAdCallback.kt
package com.rikatech.admoblibrary.callback

import com.google.android.gms.ads.AdSize
import com.google.android.gms.ads.AdValue

interface BannerAdCallback {
    fun onBannerAdLoaded(adSize: AdSize?)
    fun onAdFail()
    fun onAdPaid(adValue: AdValue?)
}
